homeworks/olena.ambitska_elena-ambitska/GamesProject/success.html

```diff
@@ -10,8 +10,10 @@
     <body>
         <main class="main login">
             <section class="success__page">
-                <button type="button" class="primary__button" onclick="history.back()">
-                    <img src="././images/arrow-left-min.svg" width="22px" height="22px" alt="go back icon" class="left__arrow"> Go Back
+                <button type="button" class="primary__button">
+                    <a href="login.html" class="back__link">
```

Comment:
It should be named `Return to homepage` and lead you to the home page, i.e. index.html
Please correct this in the next task